spell-template/config.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity >=0.5.12;
+
+contract POOL_CONFIG {
+
+    // precision
+    uint256 constant public THOUSAND = 10 ** 3;
+    uint256 constant public MILLION  = 10 ** 6;
+    uint256 constant public WAD      = 10 ** 18;
+    uint256 constant public RAY      = 10 ** 27;
+    uint256 constant public RAD      = 10 ** 45;
+
+    // Maker changelog 
+    address constant MAKER_CHANGELOG = 0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F;
+
+    // Tinlake addresses FF1 
+    address constant ROOT = 0x68CA1a0411a8137d8505303A5745aa3Ead87ba6C;
+    address constant DROP = 0x0f763b4d5032f792fA39eE54BE5422592eC8329B;
+    address constant MGR = 0x17E5954Cdd3611Dd84e444F0ed555CC3a06cB319;
```

Comment:
AFAICT, this is just a template file, but it includes parts from other real collaterals.  Perhaps just copy one of those to this file to make a better template.  The `MGR` here and the one below for `RWA004` have different addresses, but they would be the same in a normal config (like in the `RWA004` config).

Reply:
This is also fixed!